src/Services/QueuedJobLogger.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symbiote\QueuedJobs\Services;
+
+use Psr\Log\AbstractLogger;
+use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
+
+/**
+ * Writes log output to a job descriptor
+ */
+class QueuedJobLogger extends AbstractLogger
+{
+    /** @var QueuedJob */
+    public $job;
```

Comment:
Do we actually need these class properties to be public?

It's usually a thumbs-up rule to give as strict access as possible by default and only open things when it's really necessary. Otherwise, we introduce it to the public APIs which we'll have to support for the whole lifecycle of the module and won't be able to break it until the next major version release.

Reply:
Yep agreed. These can both be protected with a getter for external access